selfdrive/controls/controlsd.py

```diff
@@ -777,6 +777,7 @@ def publish_logs(self, CS, start_time, CC, lac_log):
     controlsState.startMonoTime = int(start_time * 1e9)
     controlsState.forceDecel = bool(force_decel)
     controlsState.canErrorCounter = self.can_rcv_timeout_counter
+    controlsState.experimentalMode = self.params.get_bool("ExperimentalMode") and self.CP.openpilotLongitudinalControl
```

Comment:
Hi @adeebshihadeh,
I'm reading this code and I noticed you have params.get_bool here, I'm curious wouldn't this make lots of file I/O access?

cheers,

Rick